oer-bookr/src/containers/BookDescription.js

```diff
@@ -1,16 +1,62 @@
 import React, { Component } from "react"
 
+import { DescriptionWrapper } from "../styles/bookStyles"
+import { ReviewsWrapper } from "../styles/reviewStyles"
+import Review from "../components/review/Review"
+
 class BookDescription extends Component {
   constructor(props) {
     super(props)
-    this.state = {}
+    this.state = {
+      category: "Engineering",
+      title: "About Accuracy and Approximation",
+      author: "Ronald Poveda",
+      description:
+        "Students learn about the concepts of accuracy and approximation as they pertain to robotics, gain insight into experimental accuracy, and learn how and when to estimate values that they measure. Students also explore sources of error stemming from the robot setup and rounding numbers.",
+      publisher: "TeachEngineering",
+      license: "",
+      reviews: [
+        {
+          id: 0,
+          reviewer: "Anonymous",
+          review:
+            "Students follow specific procedures to complete much of the work. Using this as an assessment may yield faulty information."
+        },
+        {
+          id: 1,
+          reviewer: "L.J",
+          review:
```

Comment:
I might call this `text` rather than `review` since this is a property on a review rather than being the review itself (in the context of the code with an array of objects called `reviews`, where each object would be a review).